ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -776,13 +766,13 @@ public class BottomSheetController: UIViewController {
 
         // Minimum padding from top when the sheet is fully expanded
         static let minimumTopExpandedPadding: CGFloat = 25.0
```

Comment:
Nit: for the sake of consistency - all CGFloat's should have .0 at the end like this one (25.0) or remove the .0 to be consistent with the rest of the file.